domain/tutoringModel/LearningPath.py

```diff
@@ -0,0 +1,102 @@
+class LearningPath:
```

Comment:
We now have two classes in the tutoring folder for the Learning Path. Please delete the model.py

Reply:
You wrote this class, I didn't know you wrote this class.

Reply:
You wrote the class model.py, I didn't know you wrote this class.